servers/biometric_signal_sensor_interface/src/hexoskinHelper/utilityHelper.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import, division, print_function
+import sys
+
+__author__ = 'abhijith'
+
+'''
+utilityHelper provides all methods that retrieves the non-biometric data
+such as user information, available users, available biometric resources, etc.
+This helper module is called by the Tango server module providing a clear
+line of abstraction.
+'''
+
+
+def allUsers():
```

Comment:
Python coding style requires function's and method's names to be snake_case, and class names to be CamelCase